### ISSUE TITLE
lavis/tasks/base_task.py accum_grad_iters division for reduction

### ISSUE THREAD

**Issue by kondvit** [salesforce/LAVIS#225]
Hello, 

First of all, awesome work Salesforce team. 

I am finetuning Blip2FlanXL model.

I noticed in base_task.py you are not dividing loss by accum_grad_iters.

https://github.com/salesforce/LAVIS/blob/47e0f3f25ca763975738c7224c8369207812ce6c/lavis/tasks/base_task.py#L218-L230

For FlanXL the default loss reduction is 'mean'. 

https://github.com/salesforce/LAVIS/blob/47e0f3f25ca763975738c7224c8369207812ce6c/lavis/models/blip2_models/modeling_t5.py#L1752

Should I divide the loss by accum_grad_iters?

`            if use_amp:
                scaler.scale(loss).backward() / accum_grad_iters 
            else:
                loss.backward() / accum_grad_iters `

Thank you

**Comment by zoujx96** [comment]
+1
